gtsfm/common/gtsfm_data.py

```diff
@@ -165,6 +180,7 @@ def add_track(self, track: SfmTrack) -> bool:
             i, _ = track.measurement(j)
 
             if i not in self._cameras:
+                # TODO (travisdriver): Should we throw an error here?
```

Comment:
TODO: should we maybe keep the track without this measurement?

Reply:
If the track is referencing a measurement that doesn't exist, I think we should throw an error instead of adding the corrupted track. But I'm open to see what others think @ayushbaid @johnwlambert